### ISSUE TITLE
PluginsServiceTests fail on windows (again)

### ISSUE THREAD

**Issue by tvernum** [elastic/elasticsearch#28758]
Related: https://github.com/elastic/elasticsearch/issues/28095

It seems that recent changes to the PluginsServiceTests have (re)introduced a windows incompatibility.

https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+6.x+multijob-windows-compatibility/700/console
```
05:55:22 ERROR   0.00s J1 | PluginsServiceTests (suite) <<< FAILURES!
05:55:22    > Throwable #1: java.io.IOException: Could not remove the following files (in the order of attempts):
05:55:22    >    C:\Users\jenkins\workspace\elastic+elasticsearch+6.x+multijob-windows-compatibility\server\build\testrun\test\J1\temp\org.elasticsearch.plugins.PluginsServiceTests_85B3DAB5DD409299-001\tempDir-037\plugins\fake-meta\fake\plugin.jar: java.nio.file.FileSystemException: C:\Users\jenkins\workspace\elastic+elasticsearch+6.x+multijob-windows-compatibility\server\build\testrun\test\J1\temp\org.elasticsearch.plugins.PluginsServiceTests_85B3DAB5DD409299-001\tempDir-037\plugins\fake-meta\fake\plugin.jar: The process cannot access the file because it is being used by another process.
05:55:22    >    C:\Users\jenkins\workspace\elastic+elasticsearch+6.x+multijob-windows-compatibility\server\build\testrun\test\J1\temp\org.elasticsearch.plugins.PluginsServiceTests_85B3DAB5DD409299-001\tempDir-037\plugins\fake-meta\fake: java.nio.file.DirectoryNotEmptyException: C:\Users\jenkins\workspace\elastic+elasticsearch+6.x+multijob-windows-compatibility\server\build\testrun\test\J1\temp\org.elasticsearch.plugins.PluginsServiceTests_85B3DAB5DD409299-001\tempDir-037\plugins\fake-meta\fake
```

It seems that this is triggered by the changes in 106f5c5925059f4655b9c2abadcdbb18b4da4180

**Comment by jasontedor** [comment]
Thanks @tvernum, I am looking into this.

**Comment by jasontedor** [comment]
Closed by 3dfb4b8b18d692369643e0f4a423c5784ed2397c.
